src/common/device/AndroidDevice.ts

```diff
@@ -19,12 +20,12 @@ Messages.importMessagesDirectoryFromMetaUrl(import.meta.url);
 const messages = Messages.loadMessages('@salesforce/lwc-dev-mobile-core', 'common');
 
 export enum AndroidOSType {
-    googleAPIs = 'google_apis',
-    googlePlayStore = 'google_apis_playstore',
-    androidDesktop = 'android-desktop',
-    googleTV = 'google-tv',
-    androidWear = 'android-wear',
-    androidAutomotive = 'android-automotive'
+    googleAPIs = 'google apis',
```

Comment:
Leave it to Google to mess this up. When querying AVDs on different platforms, some of these come back with _ and some with - as work separators. To I updated the logic during parsing to just replace all _ and - with blank space and then we can do a consistent string check on different platforms and for different OS types